Epam.TestAutomation/Epam.TestAutomation.BDD/DataDrivenTestingTests/SearchTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Utilities.Logger;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Epam.TestAutomation.Helper;
+using Epam.TestAutomation.Utils;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using Epam.TestAutomation.BDD.DataDrivenTestingTests;
+
+namespace Epam.TestAutomation.BDD
```

Comment:
Create BaseTest and move OneTimeSetUp, TearDown to it. And Create Setup to open browser and navigate to the site.